README.md

```diff
@@ -34,21 +34,23 @@ Example:
 
 ```js
 // ... in your OTEL setup
-const FastifyInstrumentation = require('@fastify/otel');
+const FastifyOtelInstrumentation = require('@fastify/otel');
 
-const fastifyInstrumentation = new FastifyInstrumentation();
-fastifyInstrumentation.setTraceProvider(provider)
+// If serverName is not provided, it will fallback to OTEL_SERVICE_NAME
+//   as per https://opentelemetry.io/docs/languages/sdk-configuration/general/.
```

Comment:
```suggestion
// If serverName is not provided, it will fallback to OTEL_SERVICE_NAME
// as per https://opentelemetry.io/docs/languages/sdk-configuration/general/.
```